### ISSUE TITLE
auto-feedback

### ISSUE THREAD

**Issue by krockpot** [MrPickles/dtl#6]
shit bot trigger should open a github issue with the text of the message. for example:
- shit bot doesn't have typo correction
- shit bot doesn't even have a countdown timer

these messages would open a github issue with the contents of the trigger message as the context.

**Comment by MrPickles** [comment]
The tricky part about having the bot open an issue is that it'll need to interact with the GitHub API. I suppose I can have it open issues up in my name, but that's prone to abuse.

As a result, I think it'll need a few extra parts to have this be a reasonable feature:

* There needs to be a way to give an API key to the bot via chat.
* There should also be a way to specify the contents of the issue description.
* The bot should wait for some sort of confirmation to avoid spam.

**Comment by krockpot** [comment]
Yeah agreed this feature request requires further discussion. I think the bot should have its own API credentials, consumed via a config file or environment variable or something else at runtime.

Maybe the bot could prompt the requestor for a more detailed description and accept their next message (with confirmation)?
